recipes/libsodium/all/conanfile.py

```diff
@@ -118,7 +108,13 @@ def source(self):
 
     @property
     def _msvc_sln_folder(self):
-        if self.settings.compiler == "Visual Studio":
+        if self.settings.compiler == "msvc":
```

Comment:
I assume this works because this function is never called with mac/linux compilers 🤔

Reply:
you are right 🙃